arches/app/models/resource.py

```diff
@@ -189,22 +189,31 @@ def bulk_save(resources):
             resource.tiles = resource.get_flattened_tiles()
             tiles.extend(resource.tiles)
 
-        # need to save the models first before getting the documents for index
+        # need to handle if the bulk load is appending tiles to existing resources/
+        existing_resources = Resource.objects.filter(resourceinstanceid__in=[resource.resourceinstanceid for resource in resources])
+        existing_resources_ids = [existing_resource.resourceinstanceid for existing_resource in existing_resources]
```

Comment:
here on line 194 I would suggest the following change to a ``set`` comprehension for performance:
``existing_resources_ids = {existing_resource.resourceinstanceid for existing_resource in existing_resources}``

Reply:
I'll change to a `set` and test

Reply:
passed testing so commited